src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -26,6 +30,11 @@ class GoogleDataprocDAO(protected val dataprocConfig: DataprocConfig)(implicit v
   private val jsonFactory = JacksonFactory.getDefaultInstance
   private val cloudPlatformScopes = List(PubsubScopes.CLOUD_PLATFORM)
 
+  private def getDataProc(applicationName:String) = {
```

Comment:
should this be a lazy val? do we need to make a new builder every time?

Reply:
I like that

Reply:
Sounds good, the applicationName appears to be constant anyway